GOVERNANCE.md

```diff
@@ -16,64 +17,125 @@ Users activities include (but are not limited to):
- 
-Therefore committers:
+Organization member have no more authorities than contributors, and they should engage with the community through the [developer discussion list](https://groups.google.com/forum/?fromgroups#!forum/openrefine-dev) or the [issue list](https://github.com/OpenRefine/OpenRefine/issues?state=open) regarding their intention. However, Organization Member have earned enough trust from the community to have direct access to the project code base without having to submit pull request. Organization Member seek approval after the contribution is made, rather than before.
+
+Therefore Organization Member:
 
 - Help contributors via the [developer discussion list](https://groups.google.com/forum/?fromgroups#!forum/openrefine-dev).
+- Manage tag, rename and organize issues, pull requests and [projects](https://github.com/OpenRefine/OpenRefine/projects)
+- Have direct access to the code base to create new branches
+- Organize the wiki
 - Merge pull requests submitted by contributors
-- Have direct access to the code base
-- Nominate and vote for new committers
 
-Committers participate in the Project Management Committee (PMC). They engage in strategic planning, release planning, and approving changes to the governance model. The list of committers is available here: https://github.com/orgs/OpenRefine/people
+Organization Member Contributors abide by the project’s [Code of Conduct](https://github.com/OpenRefine/OpenRefine/blob/master/CODE_OF_CONDUCT.md)
 
-How to become an OpenRefine Committers?  Be a contributor and be nominated as a committer. OpenRefine selects and elects new committers using the [Apache model](https://community.apache.org/newcommitter.html). You may nominate yourself. Nomination should be sent to the [developer discussion list](https://groups.google.com/forum/?fromgroups#!forum/openrefine-dev)
+#### How to be part of the OpenRefine Organization Member?  
+Be a guest contributor and be nominated as a Organization Member. Current Organization Member selects and elects new Organization Member. You may nominate yourself. Nomination should be sent to the [developer discussion list](https://groups.google.com/forum/?fromgroups#!forum/openrefine-dev)
 
+#### Current List of Organization Member 
+The list is available here: https://github.com/orgs/OpenRefine/people. 
 
-##  Decision Making Process 
+### Release Manager
+The Release Manager is responsible to 
+* Coordinate with the community to select which issue and pull request are part of the release
+* Prepare and publish the release 
```

Comment:
They might not do it themselves but use a backup or designated peer, like the current situation with Tom and Antonin, and we should not have wording that does not allow backup managers. So I suggest a more open rephrasing of:
```suggestion
* Prepare and coordinate publishing the release 
```